psl/psl-core/src/configuration/configuration_struct.rs

```diff
@@ -43,7 +43,11 @@ impl Configuration {
         })
     }
 
-    pub fn resolve_datasource_urls_from_env<F>(
+    // Resolve datasource url for query engine.
+    //
+    // The main interesting thing here is we want to ignore any error that may arise from resolving
+    // direct_url.
```

Comment:
Nitpick: this won't show up in the `cargo doc` docs with `//` comments, you have to use `///` comments instead.

Reply:
It was deliberate because I think this is a quirk/TODO, but I now see that it's actually quite relevant so maybe it should be part of its "official" documentation.

Reply:
Yep, that makes total sense then.  I've never tried it but something like this should work too:

```rust
    /// Resolve datasource url for query engine.
    //
    // The main interesting thing here is we want to ignore any error that may arise from resolving
    // direct_url.
   fn foo() { panic!() }

```